assets/stylesheets/components/_form-validation.scss

```diff
@@ -0,0 +1,29 @@
+.form-group {
+  &.has-success,
+  &.has-error {
```

Comment:
Bootstrap also provides the `.has-warning` selector. Have we considered overriding styles for that state, in addition to error and success?